plugin.video.retrospect/resources/libs/textures/remote.py

```diff
@@ -0,0 +1,53 @@
+# ===============================================================================
+# LICENSE Retrospect-Framework - CC BY-NC-ND
+# ===============================================================================
+# This work is licenced under the Creative Commons
+# Attribution-Non-Commercial-No Derivative Works 3.0 Unported License. To view a
+# copy of this licence, visit http://creativecommons.org/licenses/by-nc-nd/3.0/
+# or send a letter to Creative Commons, 171 Second Street, Suite 300,
+# San Francisco, California 94105, USA.
+# ===============================================================================
+
+from textures import TextureHandler
+
+
+class Remote(TextureHandler):
+    def __init__(self, cdn_url, logger):
+        TextureHandler.__init__(self, logger)
+
+        self.__cdnUrl = cdn_url
+        if not self.__cdnUrl:
+            self.__cdnUrl = "https://cdn.rieter.net/plugin.video.retrospect.cdn/"
+
+    def purge_texture_cache(self, channel):
```

Comment:
not allowed

Reply:
As discussed: I load my textures from a remote github.io page, because the zips became too large and crashed my shared servers. If one of those textures is update with an add-on update, Retrospect catches this and uses Kodi's JSON api to invalidate the cache for only that specific texture. That way the change is reflected immediately.

Reply:
Need a second opinion here. @Rechi ?